src/app.py

```diff
@@ -38,71 +35,67 @@
 
 if st.checkbox("Show more info about this tool"):
     notes = "The total size of the susceptible population will be the entire catchment area for Penn Medicine entities (HUP, PAH, PMC, CCH)"
-    show_more_info_about_this_tool(st=st, model=m, parameters=p, defaults=DEFAULTS, notes=notes)
+    display_more_info(st=st, model=m, parameters=p, defaults=DEFAULTS, notes=notes)
 
 st.subheader("New Admissions")
 st.markdown("Projected number of **daily** COVID-19 admissions at Penn hospitals")
-new_admit_chart = new_admissions_chart(alt, m.admits_df, parameters=p)
-st.altair_chart(
-    new_admissions_chart(alt, m.admits_df, parameters=p),
-    use_container_width=True,
+admits_chart = build_admits_chart(alt=alt, admits_df=m.admits_df, max_y_axis=p.max_y_axis)
+st.altair_chart(admits_chart, use_container_width=True)
+st.markdown(build_descriptions(chart=admits_chart, labels=p.labels))
+display_download_link(
+    st,
+    filename=("%s_projected_admits.csv" % (p.current_date,)),
```

Comment:
can this be in `.format` style please, for consistency with the rest of the codebase?

Reply:
Sure, or maybe f-strings?